visualization/view_scene.py

```diff
@@ -113,9 +95,13 @@ def view_scene(args: argparse.Namespace) -> None:
         default=0.1,
         help="if points are rendered as spheres, then spheres are rendered with this radius.",
     )
+    parser.add_argument(
+        "--frustrum_ray_len",
+        type=float,
+        default=0.3,
+        help="Length extend frustum rays away from optical center "
```

Comment:
nit: `Length to extend frustum rays ... `

Reply:
fixed